.github/workflows/automated-release.yml

```
@@ -0,0 +1,416 @@
+              echo "changelog=No release needed - no conventional commits found" >> $GITHUB_OUTPUT
+              exit 0
+              ;;
+          esac
+
+          NEW_VERSION="$MAJOR.$MINOR.$PATCH"
+          echo "New version: $NEW_VERSION"
+
+          echo "=== Generating Changelog ==="
+          CHANGELOG="## Changes in v$NEW_VERSION"$'
+
+'
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)